test/test_jit.py

```diff
@@ -6742,7 +6744,7 @@ def forward(self, x):
                 out = self.conv2d(out)
                 return out
 
-        self.checkTrace(TransformerNet(), (torch.rand(5, 3, 224, 224),))
+        self.checkTrace(TransformerNet(), (torch.rand(5, 3, 64, 64),))
```

Comment:
Just curious, why do we turn to 64 from 224?

Reply:
The test took several minutes to run. Now it still takes a minute or so.

Reply:
I see... thanks